client/src/document/baseline-indicator.tsx

```diff
@@ -7,10 +7,11 @@ import { useLocation } from "react-router";
 
 import "./baseline-indicator.scss";
 
-import type {
-  SupportStatus,
-  browserIdentifier,
-} from "../../../libs/types/web-features";
+// web-features doesn't export these types directly so we need to do a little typescript magic:
```

Comment:
Would the web-features maintainers be open to add these types?

Reply:
I'm sure they will, I'll be opening a PR to export them

Reply:
Thanks, and consider fixing their type casing at the same time, I'd be suprised if the lowercase `b` is intended.